pkg/hubble/parser/seven/http.go

```diff
@@ -115,3 +100,21 @@ func filterHeader(key string, value string, redactSettings options.HubbleRedactS
 	}
 	return value
 }
+
+// filterURL receives the observed URL and a set of Hubble redact settings and
+// conditionally redacts sensitive values.
+// If configured and user info exists, it removes the password from the flow.
+// If configured, it removes the URL's query parts from the flow.
+func filterURL(_url *url.URL, redactSettings options.HubbleRedactSettings) {
+	if _url != nil {
```

Comment:
Can we change the _url variable name to just url.  As far as I know its not common practice to prefix variable names with underscores as with other languages.

Reply:
The old code called it `uri` to avoid naming confusion

Reply:
Thank you both for the headsup on this!

> Can we change the _url variable name to just url. 

TBH I did not do this to avoid shadowing the `net/url` package that we import. For instance, see https://stackoverflow.com/questions/7772229/how-does-one-use-a-variable-name-with-the-same-name-as-a-package-in-go.

> The old code called it uri to avoid naming confusion

Indeed, I changed this because, typically, URI != URL. 

## Proposed solution

Alias `net/url` as `neturl` and freely use `url` as local variable inside functions.

Reply:
Pushed a FIXUP for this. Resolving the conversation, feel free to reopen it in case you prefer to switch back to `uri`.